ADAL/src/ADAuthenticationResult+Internal.m

```diff
@@ -155,20 +158,48 @@ + (ADAuthenticationResult*)resultForBrokerErrorResponse:(NSDictionary*)response
     
     // Otherwise parse out the error condition
     ADAuthenticationError* error = nil;
+    NSMutableDictionary* userInfo = nil;
     
     NSString* errorDetails = [response valueForKey:OAUTH2_ERROR_DESCRIPTION];
     if (!errorDetails)
     {
         errorDetails = @"Broker did not provide any details";
     }
+
+    if (response[BROKER_APP_VERSION])
+    {
+        [userInfo setValue:response[BROKER_APP_VERSION] forKey:BROKER_APP_VERSION];
```

Comment:
please also add this to public NSError header so developers know to look for it

Reply:
this file: https://github.com/AzureAD/azure-activedirectory-library-for-objc/blob/dev/ADAL/src/public/ADAuthenticationError.h